src/hotspot/share/opto/multnode.cpp

```diff
@@ -63,6 +63,16 @@ ProjNode* MultiNode::proj_out_or_null(uint which_proj) const {
   return NULL;
 }
 
+ProjNode* MultiNode::proj_out_or_null(uint which_proj, bool is_io_use) const {
```

Comment:
I considered merging them and introduce `is_io_use` as an optional argument, but was concerned that code which currently effectively doesn't care about the flavor of projection it gets (regular or IO) may fail. 
(I suspect it may be a lurking bug when it happens, but wanted to play it safe here.)

Since as you pointed out there's only a single use site, I can just inline the code there. What do you think?

Reply:
Keep it as it is. When more uses are added in the future - it might become more obvious how to make them distinct from each other.